src/components/user.js

```diff
@@ -203,17 +223,15 @@ export default class User extends React.Component {
                         disabled={!isFormFieldsValid}
                         onClick={this._handleClickSubmit}>Submit</button> :
                     (
-                        <button className="ButtonEdit"
-                                disabled={!isFormFieldsValid && editMode}
-                                onClick={editMode ? this._handleClickSave : this._handleClickEdit}>
-                            {editMode ? 'Save' : 'Edit'}
-                        </button>
+                        <Edit disabled={!isFormFieldsValid}
```

Comment:
IMO it's better to call component `EditingButtons` because it shows several buttons